mod.conf

```diff
@@ -1 +1,8 @@
 name = stamina
+author = Various
+description = Adds stamina and hunger effects.
+min_minetest_version = 5.0
```

Comment:
Just add last 5.0 supported release (git tag), bump this to 5.4 and let legacy users to use previous stable and usable version?
Allows using latest API docs without thinking about compatibility and allows fixing deprecated stuff without thinking about breaking something.

Also if needed possible bugs on old versions can still be fixed easily if someone wants to do that, it is git after all.